Buildings/Fluid/HeatExchangers/BaseClasses/WetCoilWetRegime.mo

```diff
@@ -0,0 +1,226 @@
+
+  hSatSurEffM.hSat=hSatSurEff;
+  TAirOut = TSurEff +(TAirIn  - TSurEff)*exp(-NTUAirSta);
+  QSen_flow= Buildings.Utilities.Math.Functions.smoothMin(
+    mAir_flow*cpAir*(TAirIn-TAirOut),QTot_flow,
+    delta*mWatNonZer_flow*cpWat0*5); // the last term is only for regularization with DTWater=5oC
+
+  (TAirIn-TSurAirIn)*UAAir=(TSurAirIn-TWatOut)*UAWat;
+  der(TWatOutEst)=-1/tau*TWatOutEst+1/tau*TWatOut;
+
+annotation (Icon(graphics={
+        Rectangle(
+        extent={{-100,100},{100,-100}},
+        lineColor={28,108,200},
+        fillColor={170,213,255},
+        fillPattern=FillPattern.Solid)}), Documentation(revisions="<html>
+<ul>
+<li>Jan 21, 2021, by Donghun Kim:<br/>First implementation of the fuzzy model. 
+See 
+<a href=\"https://github.com/lbl-srg/modelica-buildings/issues/622\">
+issue 622</a> 
+for more information. 
+</li>
+</ul>
+</html>", info="<html>
+<p>This model implements the calculation for a 100% wet coil. </p>
+<p>The equations from Braun (1988) and Mitchell and Braun (2012a and b),
+which are essentially the extension of the &epsilon;-NTU approach to
+simultaneous sensible and latent heat transfer, are utilized. </p>
+<p>The mathematical equations are analogous to that of the sensible heat exchanger.
+The key change is that the heat transfer is driven by an enthalpy difference
+not by an temperature difference.
+This change in the driving potential results in redefining capacitances and
+heat transfer coefficients accordinlgy.</p>
+<p>The total heat transfer rate is expressed as</p>
+<p align=\"center\">Q<sub>tot</sub>=&epsilon;* C*<sub>min </sub>
+(h<sub>air,in</sub>-h<sub>sat</sub>(T<sub>wat,in</sub>)),</p>
+<p>where &epsilon;*=f(Cr*,NTU*) and f is the same epsilon-NTU relationships
+(depending on the heat exchanger configuration) for the sensible heat exchanger.</p>
+<p>h<sub>air,in</sub> and h<sub>sat</sub>(T<sub>wat,in</sub>) are the specific enthalpies
+of the incoming moist air and saturated moist air at the water inlet temperature.</p>
+<p>The capacitances of water and air streams are defined as</p>
+<p align=\"center\">C*<sub>air</sub>=m<sub>air</sub> and
+C*<sub>wat</sub>=m<sub>wat</sub>c<sub>p,wat</sub>/csat,</p>
+<p>where csat is an specific heat capacity, which indicates the sensitivity
+of the enthalpy of the staturated moist air w.r.t. the temperature, and is defined
+here as csat=(h<sub>sat</sub>(T<sub>wat,out</sub>)-h<sub>sat</sub>(T<sub>wat,in</sub>))
+/(T<sub>wat,out</sub>-T<sub>wat,in</sub>). </p>
+<p>The capacitance ratio and minimum capacitance are naturally defined as</p>
+<p align=\"center\">Cr*=min(C*<sub>air</sub>,C*<sub>wat</sub>)/max(C*<sub>air</sub>,C*<sub>wat</sub>)
+and C*<sub>min</sub>=min(C*<sub>air</sub>,C*<sub>wat</sub>).</p>
+<p><br/>The number of transfer unit for the wet-coil is defined as NTU*=UA*/C*<sub>min</sub>, where </p>
+<p align=\"center\">UA*=1/(1/(UA<sub>air</sub>/c<sub>p,air</sub>)+1/(UA<sub>wat</sub>/csat). 
```

Comment:
Please reformat. Put variables into italic `<i>xxxx</i>` and make new line after `<p>` or `</p>`. We typically read and edit such documentation many times, and this is much easier if it is reasonably formatted. Please see other models for an example

Reply:
Revised accordingly.